.gitignore

```diff
@@ -16,7 +16,7 @@
 .#*
 .*.swp
 .DS_Store
-.cache
```

Comment:
ah, duplicate indeed

Reply:
I agree. But also nice to have this, I think, for people who don't do that.